packages/miniflare/src/plugins/assets/schema.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from "zod";
+import { PathSchema } from "../../shared";
+
+export const AssetsOptionsSchema = z.object({
+	workerName: z.string().optional(),
+	assetsPath: PathSchema.optional(),
```

Comment:
as per the interface definition below, `assetsPath` is mandatory. So this should be `z.string()` instead. This will ensure that incorrect asset options will fail zod validation and will result in an error, which is what we want. So basically AssetOptions is optional on the Miniflare options type def, but `assetsPath` should be mandatory, if those options are passed

Reply:
the validation err we'll see once `assetsPath` is marked as non-optional is this
<img width="571" alt="Screenshot 2024-08-29 at 10 08 17" src="https://github.com/user-attachments/assets/46475640-a045-4aa2-a39c-d767d4767aed">

Reply:
whoops yeah good point - I've nested the options now so the whole assetOptions object can be optional, but some of the properties required